DataManagementSystem/scripts/dirac-dms-replicate-and-register-request.py

```diff
@@ -33,18 +32,16 @@ def getLFNList( LFNs ):
   args = Script.getPositionalArgs()
 
   requestName = None
-  LFNs = None
   targetSEs = None
   if len( args ) < 3:
     Script.showHelp()
-    DIRAC.exit( 0 )
-  else:
-    requestName = args[0]
-    LFNs = args[1]
-    targetSEs = list( set( [ targetSE.strip() for targetSE in args[2:] ] ) )
+    DIRAC.exit( 1 )
+
+  requestName = args[0]
+  lfnList = getLFNList( args[1] )
+  targetSEs = list( set( [ se for targetSE in args[2:] for se in targetSE.split( ',' ) ] ) )
```

Comment:
yes, covering the case when the user doesn't remember whether there should be a space-separated list of arguments of comma-separated. I agree that a mixture is unlikely, but in DIRAC there is no real rule for space and comma, so....